src/ops/dataflow.rs

```diff
@@ -25,9 +26,30 @@ pub(crate) trait DataflowOpTrait {
     ///
     /// If not None, a single extra output multiport of that kind will be
     /// present.
+    #[inline]
     fn other_output(&self) -> Option<EdgeKind> {
         Some(EdgeKind::StateOrder)
     }
+
+    /// The edge kind for a single constant input of the operation, not
+    /// described by the dataflow signature.
+    ///
+    /// If not None, an extra input port of that kind will be present after the
+    /// dataflow input ports and before any [`DataflowOpTrait::other_input`] ports.
+    #[inline]
+    fn static_input(&self) -> Option<EdgeKind> {
+        None
+    }
+
+    /// The edge kind for a single constant output of the operation, not
+    /// described by the dataflow signature.
+    ///
+    /// If not None, an extra output port of that kind will be present after the
+    /// dataflow output ports and before any [`DataflowOpTrait::other_output`] ports.
+    #[inline]
+    fn static_output(&self) -> Option<EdgeKind> {
+        None
+    }
```

Comment:
no dataflow operations have a static output right, so this method doesn't need to exist